### PR TITLE
"E" Hardfork Meta CIP

### PR DESCRIPTION
**Pull request by trianglesphere**

### CODE REVIEW

File: CIPs/e_hf_meta_cip.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+From London
+* [EIP-1559: Fee market change for ETH 1.0 chain](https://eips.ethereum.org/EIPS/eip-1559)
+* [EIP-3198: BASEFEE opcode](https://eips.ethereum.org/EIPS/eip-3198)
+* [EIP-3529: Reduction in refunds](https://eips.ethereum.org/EIPS/eip-3529)
+* [EIP-3541: Reject new contracts starting with the 0xEF byte](https://eips.ethereum.org/EIPS/eip-3541)
+
+Modifications to the EIPs
+* EIP-3554: Difficulty Bomb Delay (Included in London) will not be included in the "E" Hardfork
+* EIP-1559 is already partially implemented on Celo but the upstream implementation does not match what is on Celo.
+* EIP-2929 may have different gas costs
```
Comment (YazzyYaz) [inline CIPs/e_hf_meta_cip.md]: can you add links to all the EIPs that won't be included in E?

File: CIPs/cip-0041.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
+---
+cip: 0041
+title: E Hardfork
+author: Joshua Gutow <@trianglesphere>
+discussions-to: TBD
```
Comment (YazzyYaz) [inline CIPs/cip-0041.md:5]: Needs a discussion link

File: CIPs/cip-0041.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+status: Proposed
```
Comment (YazzyYaz) [inline CIPs/cip-0041.md:6]: must be Draft status

File: CIPs/cip-0041.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+* [EIP-2930: Optional access lists](https://eips.ethereum.org/EIPS/eip-2930)
+
+From London
+* [EIP-1559: Fee market change for ETH 1.0 chain](https://eips.ethereum.org/EIPS/eip-1559)
+* [EIP-3198: BASEFEE opcode](https://eips.ethereum.org/EIPS/eip-3198)
+* [EIP-3529: Reduction in refunds](https://eips.ethereum.org/EIPS/eip-3529)
+* [EIP-3541: Reject new contracts starting with the 0xEF byte](https://eips.ethereum.org/EIPS/eip-3541)
+
+Modifications to the EIPs
+* EIP-3554: Difficulty Bomb Delay (Included in London) will not be included in the "E" Hardfork
```
Comment (YazzyYaz) [inline CIPs/cip-0041.md:54]: Please add links to all EIPs that won't be included in this hardfork for easier reference

File: CIPs/cip-0041.md [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+* EIP-1559 is already partially implemented on Celo but the upstream implementation does not match what is on Celo.
+* EIP-2929 may have different gas costs
+* The `BASEFEE` opcode should return the result of calling `getGasPriceMinimum` to the `GasPriceMinimum` smart contract at the end of the previous block.
+
+CIPs to be Included
+* CIP-???: Block Context
```
Comment (YazzyYaz) [inline CIPs/cip-0041.md:60]: are the CIPs added yet as a PR?

Comment (trianglesphere) [reply CIPs/cip-0041.md:60]: These CIPs are not written up, but we do intend to include them

**Review by YazzyYaz — APPROVED** [review]

**Pull request merged**